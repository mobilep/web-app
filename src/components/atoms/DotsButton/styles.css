svg.DotsButton-icon {
	width: 30px;
	height: 30px;
	transform: translateY(10px);
}
