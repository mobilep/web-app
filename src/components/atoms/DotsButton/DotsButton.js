import React from 'react';
import PropTypes from 'prop-types';
import { ButtonIcon } from '..';
import { DotsIcon } from '../../../icons';

import './styles.css';

const DotsButton = ({ className, ...rest }) =>
	<ButtonIcon className={className} {...rest} icon={<DotsIcon className='DotsButton-icon' />} />;

DotsButton.propTypes = {
	className: PropTypes.string,
};

export default DotsButton;

