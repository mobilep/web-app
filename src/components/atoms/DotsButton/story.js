import React from 'react';
import { storiesOf } from '@storybook/react';

import DotsButton from './DotsButton';

storiesOf('1. Atoms / DotsButton', module)
	.add('dots button', () => <DotsButton />);
