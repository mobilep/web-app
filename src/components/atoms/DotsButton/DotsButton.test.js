import React from 'react';
import { shallow } from 'enzyme';

import DotsButton from './DotsButton';

describe('Atom - DotsButton', () => {

	it('Should render', () => {
		const wrapper = shallow(<DotsButton />);
		expect(wrapper).toMatchSnapshot();
	});

});

