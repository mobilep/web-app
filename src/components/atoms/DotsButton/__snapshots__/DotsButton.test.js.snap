// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - DotsButton Should render 1`] = `
<ButtonIcon
  icon={
    <DotsIcon
      className="DotsButton-icon"
    />
  }
/>
`;
